AElf.OS.Network.Grpc/AuthInterceptor.cs

```diff
@@ -0,0 +1,30 @@
+using System.Threading.Tasks;
+using AElf.OS.Network.Infrastructure;
+using Grpc.Core;
+using Grpc.Core.Interceptors;
+
+namespace AElf.OS.Network.Grpc
+{
+    public class AuthInterceptor : Interceptor
+    {
+        private readonly IPeerPool _peerPool;
+
+        public AuthInterceptor(IPeerPool peerPool)
+        {
+            _peerPool = peerPool;
+        }
+
+        public override Task<TResponse> UnaryServerHandler<TRequest, TResponse>(TRequest request, ServerCallContext context, UnaryServerMethod<TRequest, TResponse> continuation)
+        {
+            if (context.Method != "/" + nameof(PeerService) + "/" + nameof(PeerService.PeerServiceBase.Connect))
```

Comment:
looks not good

Reply:
yeah I also think it's not soo good, for now I don't have a better solution for this.